src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -70,6 +70,11 @@ public CommandResult undo(Model model) throws CommandException {
                 .build();
     }
 
+    /**
+     * Returns true if command can be run in current application state.
+     * Command can run in GROUP_INFORMATION application state.
+     * @param applicationState The given application state.
```

Comment:
```suggestion

     * @param applicationState The given application state.
```
I think there should be a whitespace above `@param`